apps/antalmanac/src/components/RightPane/CoursePane/CoursePaneButtonRow.tsx

```diff
@@ -85,13 +77,10 @@ export function ColumnToggleButton() {
         setOpen(false);
     }, [setOpen]);
 
-    useEffect(() => {
-        RightPaneStore.on('columnChange', handleColumnChange);
-
-        return () => {
-            RightPaneStore.removeListener('columnChange', handleColumnChange);
-        };
-    }, [handleColumnChange]);
+    const selectedColumnNames = useMemo(
+        () => SECTION_TABLE_COLUMNS.filter((_, idx) => selectedColumns[idx]),
```

Comment:
Nit: I think I prefer either `i` or `index` for the variable name.
```suggestion
        () => SECTION_TABLE_COLUMNS.filter((_, index) => selectedColumns[index]),
```